### PR TITLE
Correct the spelling of property to last_modified

### PR DESCRIPTION
**Pull request by arvindshmicrosoft**
##### SUMMARY

This is a minor change to fix a typo in a property name.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

azure_rm_storageblob

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Akasurde** [comment]
@arvindshmicrosoft Thanks for the PR. Could you please add a changelog entry for this change so that we can backport this change to older releases? Thanks in advance.

**Comment by Akasurde** [comment]
You can find the location of changelog directory [here](https://github.com/ansible/ansible/tree/devel/changelogs/).

**Comment by arvindshmicrosoft** [comment]
Done @Akasurde !

File: changelogs/65752-fix-azure_rm_storageblob-typo [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
```
Comment (Akasurde) [inline changelogs/65752-fix-azure_rm_storageblob-typo]: Sorry, but could you please add file extension (.yml or .yaml) to this file?

Comment (arvindshmicrosoft) [reply changelogs/65752-fix-azure_rm_storageblob-typo]: Done. Sorry that I had overlooked that!

**Comment by Fred-sun** [comment]
ready_for_review

**Pull request merged**